extract_endpoint/tests/test_post_to_endpoint.py

```diff
@@ -150,3 +151,14 @@ def test_post_stream_cli_no_url(azure_service: blob.BlockBlobService,
     check_file_in_azure(azure_service, azure_emulator_coords, endpoint.TIMESTAMP_FILENAME, sample_timestamp)
     for name, contents in zip(sample_filenames, sample_file_contents):
         check_file_in_azure(azure_service, azure_emulator_coords, name, contents)
+
+
+@pytest.mark.usefixtures('run_endpoint')
+def test_run_tl() -> None:
+    runner = testing.CliRunner()
+    result = runner.invoke(post_to_endpoint.main, args=[
+        'run_tl',
+        '--url', None
+    ])
+    assert result.exit_code == HTTPStatus.BAD_GATEWAY
```

Comment:
getting a `BAD_GATEWAY` here means that Post connected successfully to Endpoint and supplied the correct salt and signature, but Endpoint could not connect to the TL web app. Essentially, Post did what it was supposed to.